### DISCUSSION TITLE
how to practice OOP in python

### DISCUSSION THREAD

**Discussion by Karimai** [codewars/codewars.com#3152] [Q&A]
I am using codewar for a while. I mainly practice python. I would like to practice with Object Oriented Programming approach. I am wondering if there are some special Kata for such a desire or not.

**Comment by Kacarott** [comment]
You can filter for specific tags such as `Object-oriented Programming`. For example have a look at [these kata](https://www.codewars.com/kata/search/python?q=&tags=Object-oriented%20Programming&beta=false&order_by=sort_date%20desc)

**Reply by Karimai** [reply]
Great. Yes. I did not know that we could search at all. I have always just train or skip the suggested one. appreciate your answer indeed!
